crates/compiler/can/Cargo.toml

```diff
@@ -21,6 +21,8 @@ bumpalo.workspace = true
 static_assertions.workspace = true
 bitvec.workspace = true
 
+ven_pretty = { path = "../../vendor/pretty" }
```

Comment:
we should prefer workspace dependencies (like the `bitvec.workspace = true` there. Makes it easier to ever change that dependency